python/lsst/fgcmcal/fgcmFitCycle.py

```diff
@@ -1084,6 +1084,17 @@ def _fgcmFitCycle(self, camera, handleDict, config=None):
         else:
             _config = self.config
 
+        # We need to validate a few configs here after we have the
+        # camera, to ensure dumb things aren't set.
+        if _config.ccdGrayFocalPlaneFitMinCcd > len(camera):
+            raise ValueError(
+                "Cannot set ccdGrayFocalPlaneFitMinCcd greater than the number of camera detectors.",
+            )
+        if _config.minCcdPerExp > len(camera):
+            raise ValueError(
+                "Cannot set minCcdPerExp greater than the number of camera detectors.",
+            )
+
```

Comment:
Numbers added.  I don't want to change the meanings of these things because it's a fraction of what (total or science ccds)?  But this is just a "sanity check" to make sure things are not crazy.